.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+
+install:
+    - python setup.py develop
```

Comment:
will this run fine before requirements are installed? it would be better to move it to the end